CMake/test_labels.cmake

```diff
@@ -0,0 +1,15 @@
+
+FUNCTION( ADD_TEST_LABELS TEST_NAME TEST_LABELS )
```

Comment:
The name of this function is very misleading. TEST_LABELS looks like an input parameter but actually it is for both input and output and so it must be initialized.
All the RUN_QMC_XXX functions also need a bit documentation listing input and output argument.

Reply:
Since @jtkrogel is offline, unless you see something overly bad, please create issues for subsequent revisions (descriptions, side effects, renaming,  etc.). I would like to get the dashboard usable again, and this PR is a dependency.

Reply:
OK. Let's fix it later.